.gitignore

```diff
@@ -6,7 +6,10 @@ Gemfile.lock
 downloads
 
 # node modules
-node_modules
+node_modules/
+!node_modules/@webcomponents/*.js
+!node_modules/@webcomponents/*.map
```

Comment:
Probably need to replace lines 10 and 11 with ...
```sh
# match loader, polyfills, and polyfill source maps
!node_modules/@webcomponents/webcomponentsjs/webcomponents-*
# match es5 adapter for modern browsers
!node_modules/@webcomponents/webcomponentsjs/custom-elements-es5-adapter.js
```

Any extra file that's been added from `node_modules/@webcomponents/webcomponentsjs` can be removed.